packages/wonder-blocks-dropdown/src/components/dropdown-core.tsx

```diff
@@ -961,6 +955,9 @@ class DropdownCore extends React.Component<Props, State> {
                     styles.dropdown,
                     light && styles.light,
                     isReferenceHidden && styles.hidden,
+                    {
+                        maxHeight: "var(--popper-max-height)",
+                    },
```

Comment:
**Suggestion**: It might help consistency to move this down into the Aphrodite styles.

Reply:
Good points! I'll move that to the Aphrodite styles + include a comment mentioning that comes from that file.